src/models/base.ts

```diff
@@ -3,7 +3,7 @@ import type { DetailedAsyncAPI } from '../types';
 
 export interface ModelMetadata {
   asyncapi: DetailedAsyncAPI;
-  pointer: string;
+  pointer?: string;
```

Comment:
I found some cases in which `pointer` will be undefined and had to do this. Happy to hear about alternatives.

Reply:
https://github.com/asyncapi/parser-js/pull/654/files#diff-d7ef7ffb6795ee82f8ee52244fb91a58a582e6b65a273d0ec162b0d04ae4c340R71-R73